examples/metrics/prometheus-install/strimzi-service-monitor.yaml

```diff
@@ -40,58 +32,10 @@ spec:
       targetLabel: namespace
       replacement: $1
       action: replace
-    - sourceLabels: [__meta_kubernetes_namespace]
-      separator: ;
-      regex: (.*)
-      targetLabel: kubernetes_namespace
-      replacement: $1
-      action: replace
-    - sourceLabels: [__meta_kubernetes_service_name]
-      separator: ;
-      regex: (.*)
-      targetLabel: kubernetes_name
-      replacement: $1
-      action: replace
-    - sourceLabels: [__meta_kubernetes_pod_node_name]
-      separator: ;
-      regex: (.*)
-      targetLabel: node_name
-      replacement: $1
-      action: replace
-    - sourceLabels: [__meta_kubernetes_pod_host_ip]
-      separator: ;
-      regex: (.*)
-      targetLabel: node_ip
-      replacement: $1
-      action: replace
-
-  #### job_name: node-exporter
-  - port: tcp-prometheus
```

Comment:
Who does not the scraping of the node exporter information? Or do we not use any information from node exporter in the dashboards and it is not needed?

Reply:
We don't use anything from it as far as I can see, and it was useless. I retested ALL the dashboards and they still work fine with the scraping jobs we have now. I think too much copy/paste was done in the paste getting this configuration from "somewhere" without really understanding what it was doing.